pkg/standalone/stop_windows.go

```diff
@@ -21,7 +21,12 @@ func Stop(appID string) error {
 
 	for _, a := range apps {
 		if a.AppID == appID {
-			eventName, _ := syscall.UTF16FromString(fmt.Sprintf("dapr_cli_%v", a.PID))
+			pid := fmt.Sprintf("dapr_cli_%v", a.CliPID)
+			if pid == "0" {
```

Comment:
This will not work because pid is a formatted string and not simply the integer value converted to string.

Reply:
Good catch, just fixed that.